www/docs/build.ts

```diff
@@ -0,0 +1,89 @@
+import { expect, run } from "effection";
+import * as esbuild from "https://deno.land/x/esbuild@v0.18.6/mod.js";
+
+import remarkFrontmatter from "https://esm.sh/remark-frontmatter@4.0.1";
+import remarkMdxFrontmatter from "https://esm.sh/remark-mdx-frontmatter@3.0.0";
+import remarkGfm from "https://esm.sh/remark-gfm@3.0.1";
+
+import rehypePrismPlus from "https://esm.sh/rehype-prism-plus@1.5.1";
+import rehypeAutolinkHeadings from "https://esm.sh/rehype-autolink-headings@6.1.1";
+import rehypeAddClasses from "https://esm.sh/rehype-add-classes@1.0.0";
+import rehypeSlug from "https://esm.sh/rehype-slug@5.1.0";
+import rehypeToc from "https://esm.sh/@jsdevtools/rehype-toc@3.0.2";
+
+import builder from "https://esm.sh/@mdx-js/esbuild@2.3.0";
+
+import structure from "../docs/structure.json" assert { type: "json" };
+
+const mdx = builder({
```

Comment:
THis is almost identical to my setup, the only  other one i use is [rehype-preset-minify](rehype-preset-minify)